packages/flutter_html_table/lib/flutter_html_table.dart

```diff
@@ -231,28 +246,31 @@ Widget _layoutCells(
           columni +=
               columnColspanOffset[columni].clamp(1, columnMax - columni - 1);
         }
+
+        final colspan = min(child.colspan, columnMax - columni);
+        final rowspan = min(child.rowspan, rows.length - rowi);
+
         cells.add(GridPlacement(
           columnStart: columni,
-          columnSpan: min(child.colspan, columnMax - columni),
+          columnSpan: colspan,
           rowStart: rowi,
-          rowSpan: min(child.rowspan, rows.length - rowi),
+          rowSpan: rowspan,
           child: CssBoxWidget(
             style: child.style.merge(row.style),
-            child: Builder(builder: (context) {
-              final alignment =
-                  child.style.direction ?? Directionality.of(context);
-              return SizedBox.expand(
-                child: Container(
-                  alignment: _getCellAlignment(child, alignment),
-                  child: CssBoxWidget.withInlineSpanChildren(
-                    children: [
-                      parsedCells[child] ?? const TextSpan(text: "error")
-                    ],
-                    style: Style(),
-                  ),
+            child: SizedBox.expand(
```

Comment:
This now allows nested tables I think, which is great! Can we still use a wrapper TagWrapExtension to make the table horizontally scrollable?